assets/newevaluationcss.css

```diff
@@ -98,9 +102,79 @@ th {
 
 .fieldset {
     position:center;
-    width: 70%;
+    box-sizing: border-box;
+    width: 100%;
     height: 150px;
+    margin-bottom: 40px;
+    display: inline-block;
+    list-style-type: none;
 }
+.fildsetlong {
+    height: 275px;
+}
+
 .height {
     height: 200px;
-}
\ No newline at end of file
+}
+
+.CenterTable {
```

Comment:
It is best to pick a naming convention from the start (i.e. lowercase, CapitalCase, camelCase, spinal-case etc) and stick to it throughout the code, for consistency, predictability, readability and ease of maintenance. Judging by the rest of the code in this area, this class name should be `.centertable`.